Mathlib/Data/Sym/Sym2.lean

```diff
@@ -426,6 +426,12 @@ theorem IsDiag.mem_range_diag {z : Sym2 α} : IsDiag z → z ∈ Set.range (@dia
 theorem isDiag_iff_mem_range_diag (z : Sym2 α) : IsDiag z ↔ z ∈ Set.range (@diag α) :=
   ⟨IsDiag.mem_range_diag, fun ⟨i, hi⟩ => hi ▸ diag_isDiag i⟩
 
+@[simp]
+lemma map_IsDiag_iff (f : α ↪ β) (s : Sym2 α) :
```

Comment:
I think that it's better to use `f : α → β` with `(hf : f.Injective)`. Also, please use implicit args.